package.json

```diff
@@ -6,7 +6,7 @@
     "postinstall": "webdriver-manager update",
     "lint": "tslint -c tslint.json -p tsconfig.json",
     "e2e": "yarn test:fullfunctional",
-    "test:fullfunctional": "protractor ./e2e/features.conf.js --cucumberOpts.tags='@nightly-test' --cucumberOpts.format='node_modules/cucumber-pretty'",
+    "test:fullfunctional": "protractor ./e2e/features.conf.js --cucumberOpts.tags=~'@nightly-test' --cucumberOpts.format='node_modules/cucumber-pretty'",
```

Comment:
We need to check with Srihari if `~` does not break nightly test run for e2e tests